torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -765,6 +782,23 @@ bool FunctionSignature::parse(PyObject* self, PyObject* args, PyObject* kwargs,
     append_overloaded_arg(&this->overloaded_args, self);
   }
   for (auto& param : params) {
+    std::cout << "\n\n\n param : " << param.name << " " << param.type_name() << std::endl;
+    switch(param.type_) {
+      case ParameterType::TENSOR_LIST: {
+        std::cout << "param is tensor list" << std::endl;
+        break;
+      }
+        
+      case ParameterType::FLOAT_LIST: {
+        std::cout << "param is float list" << std::endl;
+        break;
+      }
+
+      default: {
+        std::cout << "ELSE" << std::endl;
+      }
+    }
```

Comment:
in case of `resA = torch._foreach_add(a, a2)` (see 1.py, both are tensor lists), the whole output looks like this: 


-------------> fmt: TensorList tensors
-------------> type_str: TensorList
tensor list


-------------> fmt: ArrayRef<double> scalars
-------------> type_str: ArrayRef<double>
float list


-------------> fmt: TensorList tensors
-------------> type_str: TensorList
tensor list


-------------> fmt: Scalar scalar
-------------> type_str: Scalar
ELSE


-------------> fmt: TensorList tensors1
-------------> type_str: TensorList
tensor list


-------------> fmt: TensorList tensors2
-------------> type_str: TensorList
tensor list


-------------> fmt: Scalar alpha=1
-------------> type_str: Scalar
ELSE
in signature! (tuple of Tensors tensors, tuple of floats scalars)



 param : tensors tuple of Tensors
param is tensor list



 param : scalars tuple of floats
param is float list